api/sbom.proto

```diff
@@ -317,6 +317,15 @@ message ExternalReference {
   }
 }
 
+// Serializer message encapsulates additional metadata related to the original SBOM document.
+message Serializer {
+  string format = 1; // The original format of the SBOM document (e.g., CycloneDX, SPDX).
+  string hash = 2; // The original hash of the SBOM document to ensure integrity.
+  int64 size = 3; // The original size of the SBOM document in bytes.
+  string path = 4; // The original file path of the SBOM document.
+  string uri = 5; // The original URI of the SBOM document.
```

Comment:
Consolidate `path` and `uri` into single concept, `uri` can be `optional` so it's generated as `*string`

```suggestion
  // The original URI of the SBOM document.
  optional string uri = 4;
```